src/App.tsx

```diff
@@ -1,26 +1,56 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState, useContext } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { USER_ID, getTodos } from './api/todos';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { DispatchContext } from './store/todoReducer';
+import { Action } from './types/actions';
+import { ErrorMessage } from './components/ErrorMessage';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [errorMessage, setErrorMessage] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
```

Comment:
Not an issue at all - but possibly you'll find more usable `undefined`, not `null`

```suggestion
  const [tempTodo, setTempTodo] = useState<Todo>();
```
`tempTodo` is Todo | undefined

IMO this looks slightly better, though some prefer using `null`s

Reply:
I suppose previously I was required using null))